Remotion/Web/UnitTests/Core/Utilities/UrlUtilityTest.cs

```diff
@@ -444,7 +445,7 @@ public void AddParameter_ToEmptyUrl ()
     {
       string url = string.Empty;
       string parameter = "Parameter1";
-      string value = "Value1�#";
+      string value = "Value1�#";
```

Comment:
Many encoding changes here, please verify if everything still works as intended and check with @MichaelKetting

Reply:
all the tests are green so i guess everything still works as intended